src/shared/Core/Trace2Message.cs

```diff
@@ -277,6 +277,8 @@ public class ChildStartMessage : Trace2Message
     [JsonProperty("argv")]
     public IList<string> Argv { get; set; }
 
+    public double ElapsedTime { get; set; }
```

Comment:
Where is this set? Is it always `0`?

Reply:
It's set in `WriteChildStart` in `Trace2.cs`.